api/api.go

```diff
@@ -343,6 +343,10 @@ type CloudBackupGenericRequest struct {
 	CredentialUUID string
 	// All if set to true, backups for all clusters in the cloud are processed
 	All bool
+	// StatusFilter indicates backups based on status
+	StatusFilter CloudBackupStatusType
+	// TagFilter indicates backups based on tag
```

Comment:
Not sure if we need tag here, it is an implementation detail in portworx that isn't exposed in openstorage

Reply:
Perhaps i should rename this to metadataType, to allow implementations to have their own filter based on metadata?

Reply:
Changed this to metadata filters.